src/Components/Login.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import Box from '@mui/material/Box'
+import TextField from '@mui/material/TextField'
+import Button from '@mui/material/Button'
+import Card from './Card.js'
+import './Login.scss'
+
+const Login = () => {
+
+    const [isUsernameError, setIsUsernameError] = useState(false)
```

Comment:
Dont worry about all these states. It's okay to have so many state-controlled variables. Although we can make this code a bit efficient to reduce some lines. Here's how:

We declare one single object in state as so: 
```
const [signupFormData, setSignupFormData] = useState({
    username: '',
    usernameError: false,
   ....etc
})
```
Now we create a common state updater function: 
```
const updateFormData = event => {
    const { name, value } = event.target
    setSignupFormData({...signupFormData, [name]: value})
}
```
Try to understand how ^ works. This can reduce change handlers to 1 function.